lib/data/models/mappers/list_mapper.dart

```diff
@@ -3,6 +3,6 @@ mixin MapperList {
     dynamic originalList,
     T Function(dynamic json) listMapper,
   ) {
-    return (originalList as List<dynamic>).map((e) => listMapper(e)).toList();
+    return (originalList as List<dynamic>).map((dynamic e) => listMapper(e)).toList();
```

Comment:
doesn't this contradict with avoid_types_on_closure_parameters ?

Reply:
Yes but I did add `implicit-dynamic` to false, so when we are using `dynamic` type we are very clear about it.

Reply:
we don't need this! do we?

Reply:
Dealing with dynamic types can result in errors since the type is not known until runtime, so making it clear that this is a dynamic value cloud help